pkg/reconciler/instances/istio/README.md

```diff
@@ -1,29 +1,31 @@
-# istio-configuration reconciler
+# Istio-configuration reconciler
 
 ## Overview
 
-The istio-configuration reconciler manages `istio-configuration` Kyma component.
+Istio-configuration reconciler manages the [istio-configuration](https://github.com/kyma-project/kyma/tree/main/resources/istio-configuration) Kyma component.
```

Comment:
i would go for more general approach. atm we manage latest kyma 2 release + main. i suspect in the future we will need to take care of 2 last releases + main